src/request/stores/request-detail-store.js

```diff
@@ -102,3 +105,18 @@ function getTabs(request) {
 
     glimpse.on('data.request.detail.requested', triggerRequest);
 })();
+
+// Toggle Logging Filter Level
+(function () {
+    function triggerRequest(payload) {
+        _viewModel.logging.filter = payload;
+
+        requestChanged(_viewModel);
+    }
+
+    glimpse.on('data.request.detail.logging.filter', triggerRequest);
+})();
```

Comment:
Pity that this needs to happen so "high" up... I wonder if it would be better if the component model itself should listen for this event and emit an event that the tab is listen for. The problem with this approach though is that we are short circuiting the update chain, but I'm on the fence as to how bad that is since we are working with the same data set and its just a filter applied to that data. Also it means that this store doesn't need to know about each tab that has logic like this and its only going to get worse when we introduce concepts like sorting, etc.